frappe/tests/test_bench_cmd.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+from __future__ import unicode_literals
+
+import unittest
+import os, subprocess, shutil
+import frappe, json
+
+class TestBenchCommands(unittest.TestCase):
+	def setUp(self):
+		# Create a new bench using 'bench init'
+		bench_init_cmd = ['bench', 'init', 'test-bench']
+		self.home_dir = os.path.expanduser('~')
+
+		success = subprocess.check_output(bench_init_cmd, cwd=self.home_dir)
+
+		# Get the bench path
+		self.bench_path = os.path.join(self.home_dir, 'test-bench')
+
+	def tearDown(self):
+		# Delete the bench created
+		if os.path.exists(self.bench_path):
+			shutil.rmtree(self.bench_path)
+
+	def test_drop_site(self):
+		# Create a new site on the bench
+		new_site_cmd = ['bench', 'new-site', 'test-drop.site', '--admin-password', 'admin']
+
+		if os.environ.get('TRAVIS'):
```

Comment:
You don't need this. root_password should be mentioned in the common_site_config.json of the travis's sites/common_site_config.json